test/AspNetCoreApiSample.IntegrationTests/Resources/TaskResourceTest.cs

```diff
@@ -93,7 +93,8 @@ public async Task AspNetCoreApiSampleTaskResourceUpdate_WhenResourceDoesNotExist
         }
 
         [Fact]
-        public async Task AspNetCoreApiSampleTaskResourceUpdate_WhenResourceDoesNotExistAndNewTitleIsNull_ReturnsHttpBadRequest()
+        public async Task
+            AspNetCoreApiSampleTaskResourceUpdate_WhenResourceDoesNotExistAndNewTitleIsNull_ReturnsHttpBadRequest()
```

Comment:
I don't think this change (line break) is needed for the PR

Reply:
Indeed, this line break was introduced by the code formatting. I can remove it.